requirements.txt

```diff
@@ -23,7 +23,8 @@ pygments>=2.7.4 # not directly required, pinned by Snyk to avoid a vulnerability
 https://github.com/PyTorchLightning/lightning_sphinx_theme/tarball/master#egg=pt-lightning-sphinx-theme
 sphinxcontrib-napoleon
 mock
-recommonmark
+myst_parser
+linkify-it-py
```

Comment:
I'm getting an error that sphinx_rtd_theme package is also required when trying to build. Does that make sense? If so can you add in here?

Reply:
And in setup.py too

Reply:
Yes it makes sense because "sphinx_rtd_theme" is called by the conf.py file

Reply:
So I add it in the configuration: setup.py and requirements.txt